### ISSUE TITLE
User compatibility feedback

### ISSUE THREAD

**Issue by unknownbrackets** [unknownbrackets/ppsspp-report#5]
It should be possible to collect "feedback ratings" from users about compatibility.

For the moment, it's probably not necessary to require "unique voting."  If someone games the system, it can be addressed later.

It should be game and version specific, and include normal report details (gpu, etc.)  A "level" of compatibility (3 stars?  more aren't helpful anyway) should also be included, possibly in areas (sound, graphics, gameplay?).

Important settings might be collected as well, possibly using a "configuration profile."  Specifically, buffered rendering, hacks, frameskip, gpu core, etc. (in addition to platform and gpu.)

Could even accept screenshots and comments.

-[Unknown]

**Comment by unknownbrackets** [comment]
Just wanted to let you know, it doesn't have all the features yet, but there are pages now:

http://report.ppsspp.org/game/ULUS10336_1.00

-[Unknown]

**Comment by ufdada** [comment]
Looks very neat, i think the report list a little overloaded. There are quite many duplicated entries in there (e.g. PowerVR SGX 543 ES 2.0 IMGSGX543-113.3).

I would just use the best ones for every plattform and gpu. So users know how to setup their ppsspp configuration to get the best results.

**Comment by unknownbrackets** [comment]
Right, at the moment it's including all reports.  I want a way to discover which settings are best, but also which combinations have worked for people - sometimes people want texture scaling, or etc.

It should probably rather be in a detail that expands when you click the item, though.  Too busy right now indeed.

-[Unknown]
